src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -69,6 +70,10 @@ static ConnectionProvider newConnection() {
 		return NewConnectionProvider.INSTANCE;
 	}
 
+	static PooledConnectionProvider.Builder builder(){
+		return new PooledConnectionProvider.Builder();
+	}
+
```

Comment:
Instead of `builder()` can we add something like `elastic()` & `fixed()` and they will return different builders that will have only the relevant properties?

Reply:
ok